TMGToolbox/src/XTMF_internal/multi_class_road_assignment.py

```diff
@@ -512,8 +518,8 @@ def get_attribute_name(at):
                                 else:
                                     val = 'undefined'
                             
-                                print "Primary assignment complete at %s iterations." %number
-                                print "Stopping criterion was %s with a value of %s." %(stoppingCriterion, val)
+                                print ("Primary assignment complete at %s iterations.") %number
+                                print ("Stopping criterion was %s with a value of %s.") %(stoppingCriterion, val)
```

Comment:
The format string needs to be inside of the brackets.

Reply:
fixed the issue